docs/backend/schema.ddl

```diff
@@ -247,23 +223,25 @@ Table HostFilteredSequence {
   }
 }
 
-Table Bam {
-  id INT [pk, increment]
-  entity_id INT [not null, unique, ref: - Entity.id]
-
-  // For the MVP, we will just store them, in the future, we would ship them to archives and store just a pointer.
-  // These should be Entities.
-
-  // optional field for gisaid submission, we'd like to get users to provide this.
-  sequencing_depth FLOAT
 
-  s3_bucket VARCHAR [not null]
-  s3_key VARCHAR [not null, unique]
-
-  Indexes {
-    (s3_bucket, s3_key) [unique]
-  }
-}
+// Bam files are just another kind of output from a consensus workflow; can be treated as any other entity.
+-- Table Bam {
+--   id INT [pk, increment]
+--   entity_id INT [not null, unique, ref: - Entity.id]
+--
+--   // For the MVP, we will just store them, in the future, we would ship them to archives and store just a pointer.
+--   // These should be Entities.
+--
+--   // optional field for gisaid submission, we'd like to get users to provide this.
+--   sequencing_depth FLOAT
+--
+--   s3_bucket VARCHAR [not null]
+--   s3_key VARCHAR [not null, unique]
+--
+--   Indexes {
+--     (s3_bucket, s3_key) [unique]
+--   }
+-- }
```

Comment:
where will the sequencing depth be stored?

Reply:
Agree we should keep this